src/CommunityToolkit.Aspire.Hosting.EventStore/EventStoreBuilderExtensions.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Utils;
+using CommunityToolkit.Aspire.Hosting.EventStore;
+using HealthChecks.EventStore.gRPC;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding EventStore resources to the application model.
+/// </summary>
+public static class EventStoreBuilderExtensions
+{
+    private const string DataTargetFolder = "/var/lib/eventstore";
+    private const string LogTargetFolder = "/var/log/eventstore";
+
+    /// <summary>
+    /// Adds an EventStore resource to the application model. A container is used for local development.
+    /// The default image is <inheritdoc cref="EventStoreContainerImageTags.Image"/> and the tag is <inheritdoc cref="EventStoreContainerImageTags.Tag"/>.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <param name="port">The port on which the EventStore endpoint will be exposed.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    /// <remarks>
+    /// <example>
+    /// Add an EventStore container to the application model and reference it in a .NET project.
+    /// <code lang="csharp">
+    /// var builder = DistributedApplication.CreateBuilder(args);
+    ///
+    /// var eventstore = builder.AddEventStore("eventstore");
+    /// var api = builder.AddProject&lt;Projects.Api&gt;("api")
+    ///   .WithReference(eventstore);
+    ///  
+    /// builder.Build().Run(); 
+    /// </code>
+    /// </example>
+    /// </remarks>
+    public static IResourceBuilder<EventStoreResource> AddEventStore(
+        this IDistributedApplicationBuilder builder, string name, int? port = null)
```

Comment:
```suggestion
    public static IResourceBuilder<EventStoreResource> AddEventStore(this IDistributedApplicationBuilder builder, [ResourceName] string name, int? port = null)
```

Let's avoid arbitrary line breaks.

Also added the `ResourceName` attribute to the name parameter for the analyzer

Reply:
Addressed in the last commit: https://github.com/CommunityToolkit/Aspire/pull/277/commits/7deb7d5bc32aa830d8266342076c010509f75b88